pkg/dashboard/dashboard.go

```diff
@@ -78,3 +115,60 @@ func Dashboard(opts Options) http.Handler {
 		writeTemplate(tmpl, opts, &data, w)
 	})
 }
+
+func calculateContainerCost(costPerCPUFloat float64, costPerGBFloat float64, c summary.ContainerSummary) float64 {
+	var memCost, cpuCost float64
+	if c.Limits != nil {
+		cpuCost = costPerCPUFloat * (float64(c.Requests.Cpu().Value() + c.Limits.Cpu().Value())) / 2
+		memCost = costPerGBFloat * (ConvertToGB(c.Requests.Memory().Value()) + ConvertToGB(c.Limits.Memory().Value())) / 2
+	} else {
+		cpuCost = costPerCPUFloat * float64(c.Requests.Cpu().Value())
+		memCost = costPerGBFloat * ConvertToGB(c.Requests.Memory().Value())
+	}
+	return toFixed(cpuCost+memCost, 4)
+}
+
+func calculateRecommendedCosts(costPerCPUFloat float64, costPerGBFloat float64, containerCost float64, c summary.ContainerSummary) (float64, float64) {
+	guaranteedCpuCostRecommended := costPerCPUFloat * float64(c.Target.Cpu().Value())
+	guaranteedMemCosttRecommended := costPerGBFloat * ConvertToGB(c.Target.Memory().Value())
+
+	burstableCpuCostRecommended := costPerCPUFloat * (float64(c.LowerBound.Cpu().Value() + c.UpperBound.Cpu().Value())) / 2
+	burstableMemCosttRecommended := costPerGBFloat * (ConvertToGB(c.LowerBound.Memory().Value()) + ConvertToGB(c.UpperBound.Memory().Value())) / 2
+
+	return toFixed(containerCost-(guaranteedCpuCostRecommended+guaranteedMemCosttRecommended), 4), toFixed(containerCost-(burstableCpuCostRecommended+burstableMemCosttRecommended), 4)
```

Comment:
We should flip this subtraction. I.e. if recommendedCost > containerCost, this should be a positive number.

Reply:
I updated the code. Thank you

Reply:
@hieptl did you push your code? this still looks the same to me.